spec/support/features/authentication_steps.rb

```diff
@@ -0,0 +1,12 @@
+module Features
+  module AuthenticationSteps
+    attr_reader :current_user
```

Comment:
Future thought, I be inclined to extend this helper so we get rid of these

```
    @trainee ||= create(:trainee, provider: current_user.provider)
```

```
def trainee(attributes)
    @trainee ||= create(:trainee, attributes, provider: current_user.provider)
end
```

The reason to login is to deal with these trainees

Reply:
See my new commit - wasn't planning to do it this early, but since you brought it up, might as well start now.